server_admin/topics/identity-broker/social/openshift.adoc

```diff
@@ -37,3 +37,48 @@ Use client ID and secret defined by `oc create` command to enter them back on th
 Go back to {project_name} and specify those items.
 
 Please refer to https://docs.okd.io/latest/architecture/additional_concepts/authentication.html#oauth[official OpenShift documentation] for more detailed guides.
+
+==== OpenShift 4
+
+NOTE: Prior to configuring OpenShift 4 Identity Provider, please look for proper OpenShift 4 API URL up.
```

Comment:
is "up" related to look?  
I think it is best expressed:  "please locate the correct OpenShift 4 API URL.

Reply:
Fixed, thanks!